databases/week2/cocoa_productionTABLE.sql

```diff
@@ -0,0 +1,40 @@
+
+CREATE TABLE cocoa_production (
+	`product_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	`product_name` VARCHAR(255) NOT NULL,
+	`start_date` DATETIME NOT NULL,
+	`due_date` DATETIME NULL DEFAULT NULL,
+	`product_location` FLOAT NOT NULL,
+    `price` DECIMAL(10,2) NOT NULL CHECK (price >= 0),
+     PRIMARY KEY (`product_id`,`product_name`),
+     FOREIGN KEY (`price`) REFERENCES customer(`order_id`)
```

Comment:
You are declaring that the price column should be a foreign key. I'm not really sure about why you want that to be a foreign key